controller/User/createFavList.js

```diff
@@ -1,35 +1,42 @@
 var jwt = require('jsonwebtoken')
-const models=require('../../models')
+const models = require('../../models')
+/** @description In this function user can add movie to Favourite List
+ * @param {object} req - Request object is null
+ * @param {object} res -  Reponse object with success message if success or error message if there is an error.
+ * @param {requestCallback} next - The callback that calls the error handling middleware.
+ *  @returns {Promise}
+*/
 const createFavList = async (req, res, next) => {
     try {
         const token = req.headers['access-token']
         const payload = jwt.decode(token)
         const user = await models.User.findOne({
             where: {
-                userName: payload.userName
+                userName: req.params.userName
```

Comment:
We should not use request header in all APIs where user is necessary. Instead, After Authorization add user to req object.then u can access user object in every route

Reply:
As there is no requirement i have removed that